src/metabase/driver/generic_sql.clj

```diff
@@ -156,10 +157,18 @@
 
       A map of native DB column types (as keywords) to the `Field` `base-types` they map to.
 
-   *  `sql-string-length-fn`
+   *  `string-length-fn`
 
       Keyword name of the SQL function that should be used to get the length of a string, e.g. `:LENGTH`.
 
+   *  `stddev-fn` *(OPTIONAL)*
+
+      Keyword name of the SQL function that should be used to get the length of a string. Defaults to `:STDDEV`.
```

Comment:
There probably is some way to get the length of a string with STDDEV :disappointed: